src/include/network/lan/s390.rb

```diff
@@ -17,13 +17,18 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
+require "yast"
+
 module Yast
   # Functions for accessing and handling s390 specific needs.
   module NetworkLanS390Include
     SYS_DIR = "/sys/class/net".freeze
 
-    def initialize_network_lan_s390(_include_target)
+    def initialize_network_lan_s390(include_target)
+      include_target.class.include Yast
```

Comment:
To be honest I do not like this at all, as it can mean more failures in future as YaST collide with global namespaces in some parts like String or FileUtils. I prefer to prefix properly YaST code here with `YaST::`